src/docfx/restore/RestoreGit.cs

```diff
@@ -130,24 +129,29 @@ void AddWorkTrees()
 
                             try
                             {
-                                if (existingWorkTreeFolders.TryAdd(workTreePath))
+                                if (gitSlot.Restored && Directory.Exists(workTreePath))
+                                {
+                                    // re-use existing work tree
+                                    // checkout to {headCommit}, no need to fetch
+                                    Debug.Assert(!GitUtility.IsDirty(workTreePath));
```

Comment:
remove this `Debug.Assert` since it only runs in debug mode

Reply:
The `Debug.Assert` purpose is to let us find problem in Debug model, that's what I want to do.